src/cmap/conn/mod.rs

```diff
@@ -340,6 +377,21 @@ impl Drop for Connection {
     }
 }
 
+#[derive(Clone, Debug)]
+pub(crate) struct PinHandle {
```

Comment:
ditto docstring for this type and methods.

Also, I wonder if the name for this type may be confused a bit with the `Pin` type from `std`. What do you think about a name like `PinnedConnectionHandle`?

Reply:
Added docstrings, and `PinnedConnectionHandle` it is.